### DISCUSSION TITLE
Usage with create-react-app / outside next.js

### DISCUSSION THREAD

**Discussion by brielov** [nextauthjs/next-auth#2820] [Help] [answered]
Is it possible to use next-auth/react (v4) from create-react-app? Y have a next project which includes public facing pages and an API with next-auth. I would like to authenticate from a create-react-app admin dashboard using next-auth/react but it seems it only takes the api url from NEXTAUTH_URL. It would be nice to be able to tell SessionProvider the api endpoint so it can be used from a dashboard for example.

**Comment by balazsorban44** [comment, marked as answer]
Create React App is a no-go as it is completely client-side, and it doesn't have a secure backend that we need to run most of the logic.

We plan to expand to other Next.js-like frameworks eventually #2294
